pkg/storage/stores/shipper/compactor/retention/util_test.go

```diff
@@ -66,7 +66,7 @@ var (
 					RowShards: 16,
 				},
 				{
-					From:       dayFromTime(start.Add(49 * time.Hour)),
+					From:       dayFromTime(start.Add(73 * time.Hour)),
```

Comment:
Is this just some arbitrary value?

Reply:
We want to set the date to `start + 3 days` with an extra hour for buffer. The extra buffer gets ignored because we drop the time and just use the days.